### PR TITLE
TPRUN-4846 Patch build preparation.

### PR DESCRIPTION
**Pull request by andreasmattes**

### CODE REVIEW

File: sam/derby-starter/pom.xml [hunk 1]
```diff
@@ -15,6 +16,10 @@
 
     <name>Talend ESB :: SAM :: Derby Starter</name>
 
+    <properties>
+        <reviwion>${derby-starter.version}</revision>
```
Comment (f2par0) [inline sam/derby-starter/pom.xml:20]: Typo here

**Review by f2par0 — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -762,27 +763,10 @@
...
-                <sam-service-soap.version>7.3.1.20220920</sam-service-soap.version>
-                <security-common.version>7.3.1.20220920</security-common.version>
-                <cxf-crypto-config.version>7.3.1.20220920</cxf-crypto-config.version>
-                <correlationid-policy.version>7.3.1.20220920</correlationid-policy.version>
-                <locator-soap-service.version>7.3.1.20220920</locator-soap-service.version>
-                <cxf-sts-war.version>7.3.1.20220920</cxf-sts-war.version>
-                <text-encryptor.version>7.3.1.20220920</text-encryptor.version>
-                <groovy-all.version>${groovy.version}.20220920</groovy-all.version>
-            </properties>
+                <original.version>7.3.1.R2022-09-RT</original.version>
```
Comment (f2par0) [inline pom.xml:766]: there was no 2022-10 for the 7.3 branch ?

Comment (andreasmattes) [reply pom.xml:766]: no, 2022-09 is the latest full build.

File: sam/derby-starter/pom.xml (hunk 1 above)
Comment (andreasmattes) [reply sam/derby-starter/pom.xml]: this has been fixed

**Review by f2par0 — DISMISSED** [review]
LGTM

**Pull request merged**